src/presets/nodeless.ts

```diff
@@ -1,7 +1,13 @@
 import { NodeBuiltinModules, mapArrToVal } from "../utils";
 import type { Preset } from "../types";
+import { version } from "../../package.json";
 
 const nodeless: Preset & { alias: Map<string, string> } = {
+  meta: {
+    name: "unenv:nodeless",
```

Comment:
question: this is referred to as `nodeCompat` in `defineEnv` - do we want to unify the meta and file names here too?

Reply:
Yes i plan to do via followup PRs to constantly use `nodeCompat` term. But it requires wrangler first to adopt so we won't break it..